dace/transformation/interstate/fpga_transform_state.py

```diff
@@ -62,7 +62,8 @@ def can_be_applied(graph, candidate, expr_index, sdfg, strict=False):
                     return False
 
                 # Cannot allocate FIFO from CPU code
-                if sdict[node] is None:
+                if sdict[node] is None and (not nodedesc.transient or node.data
```

Comment:
sneaky change?

Reply:
Ye, I can revert it for this PR since it's not actually used, but it's unnecessarily restrictive right now, not allowing anything with a stream to be transformed

Reply:
Reverted